### PR TITLE
Increase severity

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content/pull/8242

## Description
Contributing a new automation 'IncreaseIncidentSeverity', which optionally increases an incident's severity to the new value, but only if greater than the existing severity. This can save some extra steps in the playbook when one wants to only increase, but not decrease, incident severity, as they don't have to do a conditional check first.

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
Hey @kirbles19 Please review yml and rn

**Comment by kirbles19** [comment]
@Shellyber Done.

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - "0"
+  - "0.5"
+  - "1"
+  - "2"
+  - "3"
+  - "4"
+  description: The incident severity to increase the incident to. Can be "Unknown", "Informational", "Low", "Medium", "High", "Critical", "0", "0.5", "1", "2", "3", or "4".
+scripttarget: 0
+runonce: false
+runas: DBotWeakRole
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.yml:30]: please add here `fromversion: 5.0.0`

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Optionally increases the incident severity to the new value if it is greater than the existing severity.
+## Script Data
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | javascript |
+| Tags |  |
+| Demisto Version | 5.5.0 |
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md:9]:
```suggestion
| Demisto Version | 5.0.0 |
```

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Outputs
+---
+There are no outputs for this script.
+
+
+## Script Example
+```!IncreaseIncidentSeverity severity=High```
+
+## Context Example
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md:26]: remove empty context

File: Tests/conf.json [hunk 4]
```diff
@@ -2860,6 +2860,10 @@
             "playbookID": "Blueliv_ThreatCompass_test",
             "fromversion": "5.0.0"
         },
+        {
+            "playbookID": "IncreaseIncidentSeverity-Test",
+            "fromversion": "5.5.0"
```
Comment (Itay4) [inline Tests/conf.json:2865]:
```suggestion
            "fromversion": "5.0.0"
```

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+Optionally increases the incident severity to the new value if it is greater than the existing severity.
+## Script Data
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | javascript |
+| Tags |  |
+| Demisto Version | 5.0.0 |
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/README.md:9]:
```suggestion
| Cortex XSOAR Version | 5.0.0 |
```

**Pull request merged**